sktalk/corpus/conversation.py

```diff
@@ -1,17 +1,47 @@
+from typing import Dict
+from typing import List
+
+
 class Conversation:
     def __init__(
-        self, utterances: list["Utterance"], metadata: dict  # noqa: F821
+        self, utterances: List["Utterance"], metadata: Dict = None  # noqa: F821
```

Comment:
Ah indeed, the `typing.List` is required in Python<3.9. From Python 3.9 on, however, using the generic type (`list`) is preferred and using the modules from `typing` is deprecated ([PEP 585](https://peps.python.org/pep-0585/)).
Unfortunately, this is not backwards-compatible, so we either have to use the `typing.List` style indeed, or require Python>=3.9 for the project. For the sake of maintainability, I would prefer the latter, given that Python 3.8 was released almost four years ago.